src/client/index.js

```diff
@@ -1,17 +1,30 @@
-/* global Meteor, Mongo */
+/* global Meteor, Mongo, Package */
 
 const client = {};
+let collection;
 
 if (typeof Meteor !== 'undefined' && Meteor.isClient) {
-  const JoinCollection = new Mongo.Collection('PublishJoin');
+  collection = new Mongo.Collection('PublishJoin');
 
   client.get = function get(name) {
-    const join = JoinCollection.findOne({
+    const join = collection.findOne({
       _id: name,
     });
 
     return join && join.value;
   };
+
+  client.has = function get(name) {
+    return !!collection.findOne({
+      _id: name,
+    });
+  };
+
+  if (Package.templating) {
```

Comment:
Though it's quite helpful to register these two global helpers, it is not really a good idea.
This module is meant to be simple and small. `Package` and `templating` are external dependencies, they add unnecessary complexity to the code (in term of maintenance). Moreover the module should be neutral about whatever frontend frameworks is used.